project-loader/src/main/kotlin/ModuleAndModelMatcher.kt

```diff
@@ -36,7 +28,7 @@ class ModuleAndModelMatcher {
     /**
      * Whether the model should be included, according to include/exclude rules. Does NOT check module inclusion rules.
      */
-    fun isModelNameIncluded(modelName: SModelName): Boolean {
+    private fun isModelNameIncluded(modelName: SModelName): Boolean {
```

Comment:
Does this method need to be made private? 
I fear that someone using our code may run into errors if we make it private without annoucement.

Reply:
Nevermind, I was looking at the wrong commit! Sorry.